kubernetes/AKS/templates/AKS_pv_claim.yaml

```diff
@@ -0,0 +1,11 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: model-store-claim
+spec:
+  storageClassName: my-azurefile
```

Comment:
NIT: Please use a descriptive name

Reply:
The name of the storageclass has been changed to "persistent-volume-azurefile". The name of the PersistentVolumeClaim is the same as that in EKS/templates/efs_pv_claim.yaml, is it necessary to change it?